### PR TITLE
Frontend : HTML Style Attribute 

### LINKED ISSUE #844: Frontend : HTML Style Attribute
<hr>

## Description 📜

I want to work on making documentation on  HTML Style Attribute  with its example code and output .
Please /assign this to me.

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev (HTML, CSS, JS)

<hr>

### PR DESCRIPTION
**Pull request by Ravinayak07**
<hr>

## Description 📜

I have created a  documentation on HTML Style Attribute with coded examples and output for the syntax.
Fixes #844 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev #HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/HTML Style Attribute/HTML_Style_Attribute.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+ ```jsx
+<h1 style="color:orange;">GWOC 2021</h1>
+<h3 style="color:green;">Frontend Development With HTML and CSS.</h3>
+```
+### ***Output :***
+<img src="assets/text_color.png">
+<p>&nbsp;</p>
+
+
+### 2. ***Text-align*** : - This property sets the horizontal alignment fo the text of an HTML Element.
```
Comment (sifa123) [inline Web_Development/FrontEnd/HTML Style Attribute/HTML_Style_Attribute.md:49]: Fix the typo

**Review by sifa123 — CHANGES_REQUESTED** [review]
Explain inline style, internal stylesheet, and external stylesheet with examples.

File: Web_Development/FrontEnd/HTML Style Attribute/HTML_Style_Attribute.md (hunk 1 above)
Comment (Ravinayak07) [reply Web_Development/FrontEnd/HTML Style Attribute/HTML_Style_Attribute.md:49]: Okay mam

**Comment by Ravinayak07** [comment]
> Explain inline style, internal stylesheet, and external stylesheet with examples.

@sifa123 I have pushed the required changes mentioned by you. Kindly please review it.

**Review by sifa123 — APPROVED** [review]

**Comment by Ravinayak07** [comment]
@sifa123 ma'am, since the PR is merged, should I close the issue now?

**Comment by sifa123** [comment]
> @sifa123 ma'am, since the PR is merged, should I close the issue now?

yeah close the issue

**Comment by Ravinayak07** [comment]
> > @sifa123 ma'am, since the PR is merged, should I close the issue now?
> 
> yeah close the issue
okay Ma'am. Thank you for your guidance.

**Pull request merged**